<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accueille</class>
 <widget class="QMainWindow" name="Accueille">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accueille</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>-30</y>
      <width>731</width>
      <height>671</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="acceuil">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>ACCUEIL</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>621</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/resource/bg.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="offre">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{

border: 1px solid #707070;
border-radius: 11px;
font-family: &quot;Century Gothic&quot;;
color: #707070;

}

QPushButton:hover{
    background-color: #3F7146;
    color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>OFFRES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="employee">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{

border: 1px solid #707070;
border-radius: 11px;
font-family: &quot;Century Gothic&quot;;
color: #707070;

}

QPushButton:hover{
    background-color: #3F7146;
    color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>EMPLOYÉ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apropos">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>410</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{

border: 1px solid #707070;
border-radius: 11px;
font-family: &quot;Century Gothic&quot;;
color: #707070;

}

QPushButton:hover{
    background-color: #3F7146;
    color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>À PROPOS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabE">
     <attribute name="title">
      <string>EMPLOYE</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-40</y>
        <width>731</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/resource/background interface.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;</string>
      </property>
      <property name="text">
       <string>EMPLOYE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 81 9pt &quot;Cantarell&quot;;
color:black;</string>
      </property>
      <property name="text">
       <string>INFORMATIONS SUR L'EMPLOYÉ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>401</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="acceuilEmployee">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>540</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color:black;
border:2px solid;
border-color: rgb(44, 77, 49);</string>
      </property>
      <property name="text">
       <string>ACCUEIL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>490</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="suprimer">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>490</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>351</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Cantarell&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>NOM:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nom">
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(154, 153, 150);
background-color: rgb(255, 255, 255);
border-radius:3px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Cantarell&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>PRENOM:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="prenom">
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(154, 153, 150);
background-color: rgb(255, 255, 255);
border-radius:3px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="rightMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Cantarell&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>DATE DE NAISSANCE:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateDeNaissance">
           <property name="timeSpec">
            <enum>Qt::LocalTime</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <property name="rightMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Cantarell&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>ANNÉE D'EMBAUCHE:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="anneeDembauche"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Cantarell&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>SEXE:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="homme">
           <property name="text">
            <string>Homme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="femme">
           <property name="text">
            <string>Femme</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Cantarell&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>MATRICULE:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="matricule">
           <property name="styleSheet">
            <string notr="true">border:1px solid rgb(154, 153, 150);
background-color: rgb(255, 255, 255);
border-radius:3px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="styleSheet">
            <string notr="true">font: 8pt &quot;Cantarell&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>FONCTION</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fonction"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabO">
     <attribute name="title">
      <string>OFFRES</string>
     </attribute>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-40</y>
        <width>731</width>
        <height>671</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/resource/background interface.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;</string>
      </property>
      <property name="text">
       <string>OFFRES</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="acceuilOffre">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>550</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color:black;
border:2px solid;
border-color: rgb(44, 77, 49);</string>
      </property>
      <property name="text">
       <string>ACCUEIL</string>
      </property>
     </widget>
     <widget class="QTableView" name="formationO">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>531</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="stageO">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>531</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="etude0">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>531</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabEmpl">
     <attribute name="title">
      <string>EMPLOYEE</string>
     </attribute>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-40</y>
        <width>731</width>
        <height>681</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/resource/resource/background interface.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>EMPLOYEE</string>
      </property>
     </widget>
     <widget class="QTableView" name="formationE">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>531</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="stageE">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>531</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="acceuilOffre_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>560</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color:black;
border:2px solid;
border-color: rgb(44, 77, 49);</string>
      </property>
      <property name="text">
       <string>ACCUEIL</string>
      </property>
     </widget>
     <widget class="QTableView" name="etudeE">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>531</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuaide">
    <property name="title">
     <string>aide</string>
    </property>
    <addaction name="actionquiter"/>
   </widget>
   <addaction name="menuaide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionquiter">
   <property name="text">
    <string>quiter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
